exercises/luhn/example.cpp

```diff
@@ -0,0 +1,44 @@
+#include "luhn.h"
+
+using namespace std;
+
+namespace luhn{
+
+    bool valid(std::string const& input_str)
+    {
+
+        string reversed;
+        reversed.assign(input_str.rbegin(),input_str.rend());
+        reversed.erase(0,reversed.find_first_not_of(" "));
```

Comment:
This just makes things more complicated when you can use `counter` to check the size at the end. (you also could just iterate through the string in reverse and it would work fine, using

```
    for (auto c = input_str.rbegin(); c != input_str.rend(); c++) {
```